README.md

```diff
@@ -21,6 +21,34 @@ Moreover, with more than 20+ custom field, you can swiftly build contact forms,
 
 So weForms is an all-in-one form builder created with the global users in mind. Its minimalistic design is not only modern but also super fast and user-friendly. Everything works on a single page. Feel no delay – because your time is precious!
 
+## Development ##
+
+### Installing Dependencies ###
+Before you can use the development version of this plugin you must install the dependencies.
+
+```
+npm install
+```
+
+### Workflow ###
```

Comment:
As far as hierarchy, it's currently:

```
### Workflow ###
### Minify JS ###
```

Are th3y separate items, or is "Minify JS" a part of the Workflow?
```
### Workflow ###
#### Minify JS ####
```

Reply:
updating to be a sub section